extensions/amp-story/1.0/amp-story-consent.js

```diff
@@ -268,4 +271,41 @@ export class AmpStoryConsent extends AMP.BaseElement {
         storyConsent.vendors && isArray(storyConsent.vendors),
         `${TAG}: story-consent requires an array of vendors`);
   }
+
+  /**
+   * Sets the accept button font color to either white or black, depending on
+   * the publisher custom background color.
+   * @private
+   */
+  setAcceptButtonFontColor_() {
+    const buttonEl =
+        dev().assertElement(this.storyConsentEl_
+            .querySelector('.i-amphtml-story-consent-action-accept'));
+    const styles = computedStyle(this.win, buttonEl);
```

Comment:
does this measure (`computedStyle`) and the subsequent write (`setImportantStyles`) need to be in a resources/vsync task?

Reply:
It's done in the buildCallback, which is run a mutate vsync context, from what I know we should be fine!

Reply:
Done